src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -84,7 +91,9 @@ tests:
     - label: "Step 2b: Reboot target device(DUT)"
       PICS: PICS_USER_PROMPT
       verification: |
-          Did the DUT successfully reboot?
+          execute the below mentioned command to put DUT into a commissionable state, Pls use equivalent command on the respective DUT
```

Comment:
Wait, why?  This should be just restarting with the same storage....  There is no "commissionable" anything going on here.

Reply:
Apologies I misunderstood, reverted back these changes as this is not needed for issue. https://github.com/project-chip/connectedhomeip/issues/29331